app/src/main/java/org/kiwix/kiwixmobile/settings/CustomSwitchPreference.java

```diff
@@ -64,6 +69,19 @@ protected void onBindView(View view) {
     ViewGroup viewGroup = (ViewGroup) view;
     clearListenerInViewGroup(viewGroup);
     super.onBindView(view);
+    setGrayState(view);
+  }
+
+  private void setGrayState(View view) {
+    boolean enabled = getPreferenceManager().getSharedPreferences().getBoolean(PREF_AUTONIGHTMODE, false);
+    TextView titleView = view.findViewById(android.R.id.title);
+    TextView summaryTV = view.findViewById(android.R.id.summary);
```

Comment:
This line can be removed since this variable is only being used once.

Reply:
I beleive this makes it more readable. Maybe someone else can comment on this.